### ISSUE TITLE
Seems to interfere with Alt-Tab

### ISSUE THREAD

**Issue by reddit-is-fun** [ryanries/UniversalPauseButton#9]
I often switch between top-level windows by using the Alt-Tab key combination. I noticed that this method always freezes borderless fullscreen/windowed games that are currently paused with Universal Pause Button (unpausing works if don't leave the window). The games don't recover and have to be killed. The issue is reproducible. 

I have currently tested with Warhammer: End Times - Vermintide, PAC-MAN Championship Edition DX+, The Binding of Isaac: Rebirth and Pinball FX2. All games have a native borderless fullscreen mode.

-----

Windows 10 64-bit

**Comment by Enigma0** [comment]
Are you running Fullscreen Exclusive? Try switching to Borderless Windowed and see if that still crashes it. (or vice versa)

**Comment by reddit-is-fun** [comment]
No, borderless window practically always including every game mentioned above.

**Comment by llagerlof** [comment]
This is happening to me too. Tested with Starbound, windowed.

**Comment by 07416** [comment]
@llagerlof This is normal:

> When you hit the actual Pause key (also known as Break) on your keyboard, the program determines which window is currently in the foreground (i.e. your game's window,) and pauses it. No matter where you are in the game. Even in the middle of one of those pesky cut scenes that would otherwise be un-pausable. When you press the key again (as long as you haven't since re-focused to another window,) the game will un-pause.

I asked _if_ this can be fixed.

**Comment by StarWeaver** [comment]
You can unstick programs in this state by right-clicking them in Process Explorer and selecting 'resume', at least. Or whatever other advanced task manager you like can probably do it.

**Comment by dlardo** [comment]
I've found that if I find myself in a "the program isn't responding" loop, I can fix it by switching to another app (notepad for example) and unpausing from there.  I suspect that the pause key is getting captured by the hung process.  Windows should pick up the keypress and deliver it to UniversalPauseButton but it's not happening. Because UPB doesn't get the message, it never knows to resume the process. We would need someone who knows Windows hotkeys better to help troubleshoot the OS.

**Comment by ryanries** [comment]
I rewrote the entire app in 2023, and I think I took care of this issue just by rewriting the logic.
